### DISCUSSION TITLE
SqlAlchemy 2 support

### DISCUSSION THREAD

**Discussion by J-Kelley** [tadams42/flask-rest-jsonapi-next#3] [General]
Are there any plans to update the library to support SqlAlchemy 2.0+ in the future?

**Comment by tadams42** [comment]
I definitely want to do that eventually. 

I hadn't switched to SA 2.0 in any of my other projects that also use flask-rest-jsonapi-next yet. When time comes to do that, flask-rest-jsonapi-next will be updated too.

**Comment by tadams42** [comment]
Hi @J-Kelley , just FYI, version `0.40.0` is out and supports `SQLAlchemy 2.x`. Be sure to check the [docs](https://flask-rest-jsonapi-next.readthedocs.io/en/latest/sqlalchemy_2x_support.html#working-with-sqlalchemy-2-x-data-layers).
